http.go

```diff
@@ -340,6 +343,14 @@ func getStatus() Status {
 
 func statusHandler(w http.ResponseWriter, req *http.Request) { json.NewEncoder(w).Encode(getStatus()) }
 
+func gpsdHandler(w http.ResponseWriter, req *http.Request) {
```

Comment:
As I mentioned in one of the other responses, think to remember setting up a service for polling the position because I could not get it to work otherwise. I absolutely agree with you, the "polling" approach would be preferable. Also disabling the endpoit by default as well as logging any access to the endpoint would a good idea.

See also https://github.com/la5nta/pat/issues/132#issuecomment-534990335

Reply:
Some more thoughts on your proposals: 
1. I don't think this will work. In my setup I use a reverse proxy to make the Pat docker container accessible on the network. Therefore it doesn't matter who is accessing the endpoint, it will allways be the same IP address (the one from the reverse proxy container).

2. It is not really user friendly but I guess it would do the job. Only I wouldn't know how to implement it by myself.

Reply:
Taking 3. an 4. into account, the following would be my proposal.
```
2019/09/25 22:44:40 Unable to get frequency from rig xyz: read tcp 127.0.0.1:42300->127.0.0.1:4532: i/o timeout.
2019/09/25 22:44:40 Starting HTTP service (:8080)...

WARNING: You have enable GPSd http endpoint (gpsd_enable_http). You might expose 
         your current position to anyone who has access to the Pat web interface!

2019/09/25 22:45:08 GPS data has been accessed! Was it you?
```
The "WARNING" shows in the web interfaces log console as well. I only removed the timestamps in order to make it more eye-catching.

Let me know your thoughts.

Reply:
I like your proposal, but on second thoughts I believe it would be sufficient to warn only if the HTTP server is exposed on all interfaces.

```go
if host, _, _ := net.SplitHostPort(addr); host == "" && config.GPSd.EnableHTTP {
  // Print warning
}
```

My main concern is with unexperienced users that simply expose the Pat on all interfaces without knowing the consequences. I think the above example should cover most of those cases.

What do you think?

Also, regarding the access log. Maybe we could make it a bit more informative by changing it to something like this?
```go
host, _, _ := net.SplitHostPort(req.RemoteAddr)
log.Printf("Location data from GPSd served to %s", host)
```